packages/editor-ui/src/views/SigninView.vue

```diff
@@ -84,19 +105,32 @@ export default defineComponent({
 		}
 	},
 	methods: {
-		async onSubmit(values: { [key: string]: string }) {
+		async onSubmit(form: {
```

Comment:
would be clearer if you split this into two functions..
```suggestion
		async onEmailPasswordSubmitted() {
		    if (mfa) {
		      this.email = email;
		      this.password = password;
		      this.showMFA();
		    } else {
		       login(email, password);
		    }
		}
		async onMFASubmitted() {
		   login(this.email, this.password, mfa);
		}
```